docs/faq.rst

```diff
@@ -702,6 +702,29 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
 .. _pipeviewer: http://www.ivarch.com/programs/pv.shtml
 
 
+How can I restore from a BTRFS snapshot to the original path?
```

Comment:
still sounds btrfs specific.

how about "How can I avoid unwanted base directories getting stored into archives?"

Reply:
It started as a BTRFS-specific FAQ. But I will make it more generic.